serialization/compact/src/main/java/CompactZeroConfig.java

```diff
@@ -0,0 +1,32 @@
+import com.hazelcast.config.CompactSerializationConfig;
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import java.util.Map;
+
+/**
+ * Demonstrates how to use Compact(BETA) serialization with java only setup
+ * If all hazelcast clients and members are java, then you can use Compact format without any config.
+ */
+public class CompactZeroConfig {
+
+    public static void main(String[] args) {
+        Config config = new Config();
+        //This config is needed only during BETA phase.
+        CompactSerializationConfig compactSerializationConfig = config.getSerializationConfig().getCompactSerializationConfig();
+        compactSerializationConfig.setEnabled(true);
+        //Note that PersonDTO does not implement Serializable. In such case, following config should be added
```

Comment:
```suggestion
        //If PersonDTO were implementing Java Serializable interface, then you would need to add the following config do that the object is serialized using the Compact serialization. Otherwise, it defaults to Java serialization.
```